src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final byte OPERATION_TYPE_INDEX = 0;
+    private static final byte AMMOUNT_INDEX = 1;
+    private static final byte SUPPLY_INDEX = 0;
+    private static final byte BUY_INDEX = 1;
+    private static final String SEPARATOR = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] processedData = readFile(fromFileName);
+        writeFile(toFileName, getResultToWrite(processedData));
+    }
+
+    private int[] readFile(String fileName) {
```

Comment:
in this method you should return lines that were read from the file only, the call of the method for calculation of buy and supply should be moved to getStatistic method